lib/v2/scnu/physics_school_announcements_and_news.js

```diff
@@ -0,0 +1,91 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+const getAritlces = async (category, url) => {
+    const { data } = await got(url);
+    const $ = cheerio.load(data);
+    const spiderResult = $('ul.article-list')
+        .children()
+        .toArray()
+        .map(async (item) => {
+            item = $(item);
+            const a = item.find('a');
+            const link = a.attr('href');
+            const { data } = await got(link);
+            const aritcle_content = $(data).find('div.detail').text();
+            return {
+                title: a.text(),
+                link,
+                pubData: timezone(parseDate(item.find('span').text()), +8),
+                category,
+                description: aritcle_content,
+            };
+        });
+    if (spiderResult.length === 0) {
+        return [];
+    }
+    return Promise.all(spiderResult);
+};
+
+const getItemsFromURLs = async (URLs) => {
+    let items = [];
+
+    for (const key of Object.keys(URLs)) {
+        items.push(getAritlces(key, URLs[key]));
+    }
```

Comment:
You can use [map()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create a new array.